### PR TITLE
[PR] Accessible fields of study

### PR DESCRIPTION
**Pull request by jeremyfelt**

### CODE REVIEW

File: includes/fields-of-study.php [hunk 1]
```diff
@@ -10,29 +10,34 @@ public function display_fields_of_study( $atts ) {
...
-						<li class="fos-item"><a href="http://admission.wsu.edu/academics/fos/Public/area.castle?id=7807">Foreign Languages and Cultures</a></li>
-						<li class="fos-item"><a href="http://admission.wsu.edu/academics/fos/Public/area.castle?id=7775">Health Sciences</a></li>
-						<li class="fos-item"><a href="http://admission.wsu.edu/academics/fos/Public/area.castle?id=1862">History, Literature, and Philosophy</a></li>
-						<li class="fos-item"><a href="http://admission.wsu.edu/academics/fos/Public/area.castle?id=1861">Music</a></li>
-						<li class="fos-item"><a href="http://admission.wsu.edu/academics/fos/Public/area.castle?id=7802">Physical and Mathematical Sciences</a></li>
-						<li class="fos-item"><a href="http://admission.wsu.edu/academics/fos/Public/area.castle?id=7839">Pre-Professional Studies</a></li>
-						<li class="fos-item"><a href="http://admission.wsu.edu/academics/fos/Public/area.castle?id=1876">Social Sciences</a></li>
-						<li class="fos-item"><a href="http://admission.wsu.edu/academics/fos/Public/area.castle?id=32423">Sport and Fitness</a></li>
-					</ul>
+				<h4>Find my field of study</h4>
```
Comment (slocker3) [inline includes/fields-of-study.php]: I think that this should be a `<h3>`

File: includes/fields-of-study.php [hunk 2]
```diff
@@ -10,29 +10,34 @@ public function display_fields_of_study( $atts ) {
...
+					</select>
+					</label>
+					<input id="fos--select" value="View field of study" type="button" />
+				</form>
 			</div>
 		</div>
+
 		<div class="find-fos-alpha-wrapper">
 			<div class="find-fos-alpha">
 				<h3>Browse Alphabetically</h3>
```
Comment (slocker3) [inline includes/fields-of-study.php]: I think this makes sense under _Find My Field of Study_ so it probably should be a `<h4>`

File: includes/fields-of-study.php (hunk 1 above)
Comment (jeremyfelt) [reply includes/fields-of-study.php]: Agreed

File: includes/fields-of-study.php (hunk 2 above)
Comment (jeremyfelt) [reply includes/fields-of-study.php]: Agreed here too :)

**Pull request merged**